doc_admin/AdminManual.adoc

```diff
@@ -79,26 +79,22 @@ Open Hospital official releases can be downloaded from GitHub: https://github.co
 
 More information can be found on the Open Hospital website: https://www.open-hospital.org/download
 
-[#setup]
-=== 1.6 Initial setup
+[#unpack]
+=== 1.6 Unpack
 
-Open Hospital does not require installation. It is sufficient to unzip the downloaded package in a folder of your choice, e.g.:
+NOTE: Do not double-click on the **oh.bat** startup script without unzipping the package first!
 
-* on Linux:
-----
-/home/OH/OpenHospital-[VERSION]
-----
+Most operating systems allow to "open" a downloaded compressed archive by just double-clicking on it. 
```

Comment:
Change: `Most operating systems allow to "open"` to `Most operating systems allow one to "open"`

Reply:
ok, thanks!

Reply:
I will definitely leave "setup" and not "unpack".
I wouldn't mention a general note about the most operating system, I would just leave the instructions plain and simple. We have already removed a lot of "conversational" informations from the admin manual.

Reply:
Maybe now it's too much "formal" that few understand it, apparently... :)

IMHO, if users don't understand, the instructions are useless, no matter how "savy" and "tech" they are.

We have seen that many users double-click from inside the archive instead to unpack it, so I believe that information is very useful and relevant for our target users, also not at all "conversational". 

The instrutions to unpack a zip file from command line in Windows, it is "plain and simple" like to "screw in a bolt with a hammer" :)

Also, why to call "setup" something that is actually just unpacking? Setup what?

Reply:
> See my inline comments

I don't know why some comments are messed up, referring to different commits maybe.
Let's start over from the [Files Changed](https://github.com/informatici/openhospital-doc/pull/131/files) page.

Some comments changes applied though.